### DISCUSSION TITLE
Flashforge m 5 pro external\internal filtration fan autostart trouble

### DISCUSSION THREAD

**Discussion by RiZo09** [OrcaSlicer/OrcaSlicer#4715] [Q&A]
in version 2.0 beta and the latest on the flashforge m5 pro printer, when printing starts, the exhaust fans and the fan for the supply air from outside are forcibly turned on, in version 1.7.0 this function worked correctly, and during printing, even if the printer is turned off, they continue to work until printing stops

**Comment by 4eIIIuP** [comment]
Check the "rod settings" - > "cooling" - > "Exhaust fan" - > "On air filtration". The check mark should be unchecked

**Reply by RiZo09** [reply]
![image](https://github.com/SoftFever/OrcaSlicer/assets/110683483/73e43d1c-315b-45dd-96f5-695de7e5c7db)
this is unchecked, but always turn on when printing start

**Reply by 4eIIIuP** [reply]
In other words, is M106 P3 still present in your G code?

**Comment by RiZo09** [comment]
Start gcode

`M190 S[bed_temperature_initial_layer_single]
M104 S[nozzle_temperature_initial_layer]
G90
M83
G1 Z5 F6000
G1 E-1.5 F800
G1 X110 Y-110 F6000
G1 E2 F800
G1 Y-110 X55 Z0.25 F4800
G1 X-55 E8 F2400
G1 Y-109.6 F2400
G1 X55 E5 F2400
G1 Y-110 X55 Z0.45 F4800
G1 X-55 E8 F2400
G1 Y-109.6 F2400
G1 X55 E5 F2400
G92 E0`

End gcode
`G1 E-3 F3600
G0 X50 Y50 F30000
M104 S0 ; turn off temperature`

it's all is default

**Reply by 4eIIIuP** [reply]
I asked not only about the start and end g code, but in general about the entire file.

**Reply by RiZo09** [reply]
I don't use the extra one

**Reply by 4eIIIuP** [reply]
You can still look in the G-code through notepad. 
Also look in "Printer settings" -> "Basic information" -> "Accessory" -> "Support air filtration"
